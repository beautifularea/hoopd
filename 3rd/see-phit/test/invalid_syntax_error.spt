R"*(
<HTML>
  /DIV> 
    Missing open angle bracket
  </DIV>
</HTML>
)*"_html; 