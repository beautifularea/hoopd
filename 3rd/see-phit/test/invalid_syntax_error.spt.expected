Error() [with int ROW = 3; int COL = 7; WHAT = spt::Unexpected_character_inside_tag_content]':
